src/main/java/com/puppycrawl/tools/checkstyle/utils/CheckUtils.java

```diff
@@ -76,18 +76,15 @@ private CheckUtils() {
      * @param typeAST a type node.
      * @return {@code FullIdent} for given type.
      */
-    public static FullIdent createFullType(DetailAST typeAST) {
-        final DetailAST arrayDeclaratorAST =
-            typeAST.findFirstToken(TokenTypes.ARRAY_DECLARATOR);
-        final FullIdent fullType;
+    public static FullIdent createFullType(final DetailAST typeAST) {
+        DetailAST ast = typeAST;
 
-        if (arrayDeclaratorAST == null) {
-            fullType = createFullTypeNoArrays(typeAST);
+        // ignore (multi-dimensional) array part of type
```

Comment:
When I read this comment I start thinking that only multi-dimensional arrays are ignored. Please make it just `ignore (multi-dimensional) array part of type`.

Reply:
fixed